tests/test_config_consistency.py

```diff
@@ -569,12 +569,21 @@ def setup_main(self):
     def test_main(self):
         assert self.req.status_code == 200
 
-        runtime_metrics = [
+        runtime_metrics_gauges = [
             metric
             for _, metric in interfaces.agent.get_metrics()
             if metric["metric"].startswith("runtime.") or metric["metric"].startswith("jvm.")
         ]
-        assert len(runtime_metrics) > 0
+
+        runtime_metrics_sketches = [
+            metric
+            for _, metric in interfaces.agent.get_sketches()
+            if metric["metric"].startswith("runtime.") or metric["metric"].startswith("jvm.")
+        ]
+
+        assert len(runtime_metrics_gauges) > 0 or len(runtime_metrics_sketches) > 0
+
+        runtime_metrics = runtime_metrics_gauges if len(runtime_metrics_gauges) > 0 else runtime_metrics_sketches
 
         for metric in runtime_metrics:
             tags = {tag.split(":")[0]: tag.split(":")[1] for tag in metric["tags"]}
```

Comment:
Should we be verifying that `tags.get("lang")` matches the language we expect to see for the running test?

Reply:
Good call, I agree that we should assert what value each library emits for the `lang`. I'll update this to assert the per-language value

Reply:
Done in ec09375714c3e59d4775219995f9c6a74a929e6e